docs/card.html

```diff
@@ -87,7 +89,7 @@ <h3>Card Subtitle</h3>
     
     <div class="content">
         <p>3. Vertical Card:</p>
-        <div class="card-container vertical-card">
+        <div class="card-container">
             <img src="/pics/product/woodland shoe.jpg">
```

Comment:
your image file name should be like woodland_shoe.jpg
bcoz i think some time server not read this time file name please check and confirm